src/BackedEnum.php

```diff
@@ -23,12 +23,6 @@ public static function fromKey(string $fqn, string $keyName): \BackedEnum
      */
     public static function tryFromKey(string $fqn, string $keyName): ?\BackedEnum
     {
-        foreach ($fqn::cases() as $case) {
-            if ($case->name === $keyName) {
-                return $case;
-            }
-        }
-
-        return null;
+        return defined("{$fqn}::{$keyName}") ? constant("{$fqn}::{$keyName}") : null;
```

Comment:
One small thing: these curly brackets are not necessary:
```
return defined("$fqn::$keyName") ? constant("$fqn::$keyName") : null;
```

Great that you found such an elegant oneliner that is so much quicker!

Reply:
My heart hearts. Those braces are for readability. Without those humans are not able to parse the string.

Reply:
```suggestion
        return defined("$fqn::$keyName") ? constant("$fqn::$keyName") : null;
```